collectors/bzimport/convertors.py

```diff
@@ -246,6 +246,19 @@ def clean_affects(self):
                 # affect does not exist any more
                 old_affect.delete()
 
+    def clean_comments(self):
```

Comment:
I agree that the cleanup here is necessary to replace the previously added temporary comment but we do not need any special functionality to discover it. Simply the temporary comment has an empty `external_system_id` so here you can just remove all such comments and that should be it.